server/src/db/dal/Initialized-data.ts

```diff
@@ -0,0 +1,127 @@
+import { Category } from "../schemas/category.schema";
```

Comment:
Would create a package called `init` and rename this file to be `hard-coded-data.ts`